### ISSUE TITLE
pywal's template

### ISSUE THREAD

**Issue by aleprovencio** [alphapapa/solarized-everything-css#53]
Hello @alphapapa,
I find your project great and integrating it to `pywal` might be the ricer's bliss.

Maybe a `pywal` template out of one of the `all-sites.css` versions of `solarized-everything-css` is enough?

Do you have interest on coding it or can we base another repo from your code?

Cheers

**Comment by jgkamat** [comment]
Alexandre Provencio writes:

> Maybe a `pywal` template out of one of the `all-sites.css` versions of `solarized-everything-css` is enough?

Different themes are generated from theme folders, for example, this is the
one for solarized:

https://github.com/alphapapa/solarized-everything-css/tree/master/themes/solarized

In it, there's a file to define colors, and two files to define dark/light
themes.

I'm not sure what pywal is, but if its a color pallet generator, you probably
could generate a 'colors.styl' file with the solarized dark/light files (if
you want little control), or with custom dark/light files (if you want more
control).

**Comment by aleprovencio** [comment]
Yeah, `Pywal` is color pallet generator from images or from over 250 pre-defined themes and then applies the colors system-wide.  A command like `wal --theme base16-nord -g` would apply the `nord` theme to your WM, App Launcher, Text Editor, Taskbar, Terminal Emulator, GUI apps, etc, without touching config files.

I thought coding `solarized-everything-css` as an [User Template](https://github.com/dylanaraps/pywal/wiki/User-Template-Files), which can read the generated colors from `~/.cache/wal/colors.css`.

**Comment by aleprovencio** [comment]
ops, didn't mean to close this issue, sorry
